### PR TITLE
[DO-1808] Add english translations sections kved

### PR DESCRIPTION
**Pull request by Iva-khar**
For filling eng kved run `$python manage.py fill_eng_kved_sections`
Or run `KvedConverter().save_to_db()` if KVED DB is empty

### CODE REVIEW

File: business_register/models/pep_models.py [hunk 1]
```diff
@@ -159,10 +158,6 @@ def check_companies(self):
     def pep_org_ua_link(self):
         return 'https://pep.org.ua/uk/person/' + str(self.source_id)
 
-    @property
-    def place_of_birth_en(self):
-        return transliterate(translate_country_in_string(self.place_of_birth))
-
```
Comment (roman-tiukh) [inline business_register/models/pep_models.py:165]: ???

File: business_register/models/company_models.py [hunk 2]
```diff
@@ -120,10 +119,6 @@ def is_foreign(self):
             return None
         return self.country.name != 'ukraine'
 
-    @property
-    def address_en(self):
-        return transliterate(translate_country_in_string(self.address))
-
```
Comment (roman-tiukh) [inline business_register/models/company_models.py:126]: ???

File: business_register/converter/kved.py [hunk 3]
```diff
@@ -105,6 +108,12 @@ def save_kveds_2005_to_kved_table(self, file):
                                      is_valid=False)
             print('All kveds 2005 were saved')
 
+    @staticmethod
+    def fill_name_en_sections():
+        for section in KvedSection.objects.all():
+            section.name_en = KVED_SECTIONS.get(section.name, '')
+            section.save()
```
Comment (roman-tiukh) [inline business_register/converter/kved.py:115]: add update_fields to save method

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: business_register/models/pep_models.py (hunk 1 above)
Comment (Iva-khar) [reply business_register/models/pep_models.py]: fixed

File: business_register/models/company_models.py (hunk 2 above)
Comment (Iva-khar) [reply business_register/models/company_models.py]: fixed

File: business_register/converter/kved.py (hunk 3 above)
Comment (Iva-khar) [reply business_register/converter/kved.py]: done

**Review by roman-tiukh — CHANGES_REQUESTED** [review]
need to add new field to kved serializator

**Comment by Iva-khar** [comment]
> need to add new field to kved serializator

Added properties to `CompanyToKved` model for use in new `CompanyToKvedSerializer`
Kveds in Company API represents like:
```
"kveds": [
                {
                    "code": "41.10",
                    "name": "організація будівництва будівель",
                    "group": "організація будівництва будівель",
                    "division": "будівництво будівель",
                    "section": "будівництво",
                    "section_en": "construction",
                    "primary_kved": false
                },
                {
                    "code": "20.41",
                    "name": "виробництво мила та мийних засобів, засобів для чищення та полірування",
                    "group": "виробництво мила та мийних засобів, засобів для чищення та полірування, парфумних і косметичних засобів",
                    "division": "виробництво хімічних речовин і хімічної продукції",
                    "section": "переробна промисловість",
                    "section_en": "manufacturing",
                    "primary_kved": false
                }
]
```

File: business_register/models/company_models.py [hunk 4]
```diff
@@ -194,6 +194,48 @@ class Meta:
     def __str__(self):
         return f"{self.kved} (declared as primary)" if self.primary_kved else str(self.kved)
 
+    @property
+    def code(self):
+        if self.kved:
```
Comment (roman-tiukh) [inline business_register/models/company_models.py:199]: is kved can be None??

File: business_register/models/company_models.py [hunk 5]
```diff
@@ -194,6 +194,48 @@ class Meta:
...
+            return self.kved.section.name
+        else:
+            return None
+
+    @property
+    def section_en(self):
+        if self.kved:
+            return self.kved.section.name_en
+        else:
+            return None
```
Comment (roman-tiukh) [inline business_register/models/company_models.py:237]: move this to serializer
like:
```
section_en = serializers.CharField(source="kved.section.name_en")
```
`select_related` will be good

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: business_register/models/company_models.py (hunk 5 above)
Comment (Iva-khar) [reply business_register/models/company_models.py:237]: +

File: business_register/models/company_models.py (hunk 4 above)
Comment (Iva-khar) [reply business_register/models/company_models.py]: rewrited

File: business_register/models/company_models.py (hunk 5 above)
Comment (Iva-khar) [reply business_register/models/company_models.py]: done

**Review by roman-tiukh — APPROVED** [review]

**Review by EmmaSamoilova — APPROVED** [review]

**Pull request merged**